tests/Test/IsIdentical.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Test;
+
+use PhpCsFixer\PhpunitConstraintIsIdenticalString\Constraint\IsIdenticalString;
+use PHPUnit\Framework\Constraint\IsIdentical as PhpUnitIsIdentical;
+
+/**
+ * @internal
+ *
+ * @todo Remove me when usages will end up in dedicated package.
+ */
+trait IsIdentical
```

Comment:
let either name it or put to subnamespace of `Constraint`

Reply:
Trait renamed to `IsIdenticalConstraint`.